keyboards/0xcb/1337/config.h

```diff
@@ -17,6 +17,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 #include "config_common.h"
+#ifdef RGBLIGHT_ENABLE
+  #include <stdio.h>
+#endif
```

Comment:
```suggestion
```
This should not be needed.

Reply:
just wanted to minimize the firmware when RGB and thus the RGB modes display is deactivated, will of course remove this logic if you deem it unnecessary

Reply:
stdio.h is indirectly included regardless of whether `RGBLIGHT_ENABLE` is defined, and generally header file inclusions don't/shouldn't affect filesize.